ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/underlay-overlay/spine-router-bgp-neighbors.j2

```diff
@@ -31,4 +31,19 @@
 {%             endif %}
 {%         endfor %}
 {%     endfor %}
-{% endif %}
\ No newline at end of file
+{% endif %}
+{# Underlay network peering to super spines #}
```

Comment:
Maybe add a {% if spine['uplinks_to_super_spine_interfaces'] is defined and spine['uplinks_to_super_spine_interfaces'] is not none %} like we have everywhere else?

It should probably be around the full config section regarding super-spines